packs/infrastructure/config.yml

```diff
@@ -8,7 +8,43 @@ title: Infrastructure
 
 # Description of the pack
 description: |
-  The infrastructure pack allows you to get visibilility into the performance and availability of your hosts and operating system.
+  ## Infrastructure Monitoring Quickstart
+
+  Infrastructure monitoring provides deep visibility into performance, available resources, and your entire estate at a glance. Remote infrastructure monitoring tools collect and display data and related attributes represented by various metrics and metadata.
+
+  Monitoring helps cloud-based DevOps teams quickly identify and resolve issues, averting potential downtime. By triggering alerts, server monitoring also improves overall flexibility and scalability to manage peaks. You can drill down into specific errors and resolve them to ensure that your environment is fully optimized.
+ 
+  New Relic's infrastructure quickstart provides visibility across the entire stack and enables greater scale and efficiency. It's a monitoring solution for multifaceted hybrid environments with customized dashboards and visualizations.
+
+
+  ### What's Included?
+
+  New Relic's infrastructure monitoring tool provides instant observability out-of-the-box. 
+
+  This quickstart includes multiple customizable dashboards, including:
+
+  - Server CPU
+  - Memory usage
+  - System load
+  - Process breakdown, and more
+
+
+  #### New Relic Infrastructure Monitoring
+
+  Instant observability with New Relic quickstarts minimizes the complexity in efficiently managing enterprise infrastructure. You can see everything at a glance with New Relic's infrastructure dashboard.
+
+  The dashboards in this quickstart enable DevOps teams to quickly see key components to their infrastructure. System load dashboards give key insights into how heavy the load is on your server. It can help prevent loss of full functionality from too many system processes at a given time. 
+
+  New Relic's instant observability quickstart helps DevOps engineers reduce complexity and enhance efficiency through unmatched visibility in an infrastructure dashboard. 
+
+
+  ### What Makes New Relic Unique?
```

Comment:
Not sure we need a pitch for new relic in individual quickstarts, it seems unnecessary and adds to the length of the markdown which we want to try to be more succinct.

Reply:
I would argue we should make the pitch. My understanding is that one of the goals for I/O is to drive new user sign-ups. Assuming portion of our users will have their first experience with New Relic through quickstarts, I think it's important to highlight the value of the entire platform.